### PR TITLE
Fix TestInitialize and TestCleanup analyzers resources

### PR DESCRIPTION
**Pull request by engyebrahim**

### CODE REVIEW

**Comment by MarcoRossignoli** [comment]
why not TestInitialize?

**Review by MarcoRossignoli — COMMENTED** [review]
why not TestInitialize?

**Comment by engyebrahim** [comment]
@MarcoRossignoli its's like that in all messages in testInitialize analyzer, should i change it all?

File: src/Analyzers/MSTest.Analyzers/Resources.resx [hunk 1]
```diff
@@ -255,7 +255,7 @@
     <value>Test Initialize '{0}' should return 'void', 'Task' or 'ValueTask'</value>
   </data>
   <data name="TestInitializeShouldBeValidTitle" xml:space="preserve">
-    <value>Test methods should have valid layout</value>
+    <value>Test Initialize should have valid layout</value>
```
Comment (Evangelink) [inline src/Analyzers/MSTest.Analyzers/Resources.resx]: Thanks @engyebrahim, I totally missed that during review. I think what reads best is probably `Test initialize methods should have valid layout`. For the messages, I would go with `Test initialize method '....`.

@cvpoienaru I might have done the same mistake for cleanup, could you check please?

Comment (engyebrahim) [reply src/Analyzers/MSTest.Analyzers/Resources.resx]: @Evangelink I fixed it for both analyzers

**Review by Evangelink — APPROVED** [review]
Awesome, thanks @engyebrahim!

I have updated the PR title to better reflect the new scope of the changes.

**Pull request merged**